backend/.env.example

```
@@ -36,6 +36,31 @@ REPUTATION_SNAPSHOT_CRON=0 0 * * *
+#                               Default: every 6 hours.
+# TTL_MONITOR_TIMEZONE        – Timezone for the cron schedule.
+#                               Default: UTC.
+# TTL_MONITOR_THRESHOLD_LEDGERS – TTL in ledgers below which a bump is submitted.
+#                               Must be ≤ the contract's TTL_EXTEND_LEDGERS
+#                               (518,400 ≈ 30 days).  Default matches the
+#                               contract's TTL_THRESHOLD_LEDGERS (241,920 ≈ 14
+#                               days) so entries are refreshed before they hit
+#                               the expiration window.
+# TTL_MONITOR_BUMP_CONCURRENCY – Max concurrent RestoreReputation transactions
+#                               per monitor run.  Default: 5.
+TTL_MONITOR_CRON=0 */6 * * *
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)